docs/pages/management/guides/gcp-tags.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: GCP Labels and Tags as Teleport Node Labels
+description: How to set up Teleport Node labels based on GCP labels and tags
+h1: Sync GCP Labels/Tags and Teleport Node Labels
+---
+
+When running on an GCP Compute instance, Teleport will automatically detect and import GCP labels and tags as
+Teleport labels for SSH nodes, Applications, Databases, and Kubernetes clusters. Both tags and labels imported
+this way will have the `gcp/` prefix; additionally, tags will receive the `tag/` infix and labels will receive
+the `label/` infix. So for an instance with label `foo=bar` and tag `baz=quux`, it will have the Teleport labels
+`gcp/label/foo=bar` and `gcp/tag/baz=quux`.
+
+When the Teleport process starts, it fetches all tags and labels from 
+the GCP API and adds them as labels. The process will update the tags every hour, 
+so newly created or deleted tags will be reflected in the labels.
+
+If the label `TeleportHostname` (case-sensitive) is present, its value will override the node's hostname.
+
+```bash
+$ tsh ls
+Node Name            Address        Labels                                                                                                                  
+-------------------- -------------- -------------------------------------------------------------------------------------------
+fakehost.example.com 127.0.0.1:3022 gcp/label/testing=yes,gcp/tag/environment=staging,aws/TeleportHostname=fakehost.example.com
+```
+
+<Notice type="note">
+  For services that manage multiple resources (such as the Database Service), each resource will receive the
+  same tags and labels from GCP.
+</Notice>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+- One Teleport agent running on a GCP Compute instance. See
+  [our guides](../../agents/join-services-to-your-cluster.mdx) for how to set up Teleport agents.
+
+## Configure service account on instances with Teleport nodes
+
+Create a service account that will give Teleport the IAM permissions needed
+to import tags and labels.
+
+<Tabs>
+<TabItem label="Console">
```

Comment:
I'm finding the console instructions pretty hard to read since the screenshots blend in with the text. I think we should put the instructions in an ordered list and indent the screenshots so it's easier to tell when they belong to separate instructions.

On the other hand, would it make sense to remove the console instructions entirely and stick to `gcloud`? This would simplify the guide a lot.